easybuild/easyconfigs/s/Simpack/Simpack-2017.eb

```diff
@@ -0,0 +1,21 @@
+# Author: Alexi Rivera
+# Chalmers University of Technology
+
+name = 'Simpack'
+version = '2017'
+
+# Simpack is a commercial software
+homepage = 'http://www.simpack.com'
+
+description = """ SIMPACK is a general purpose Multi-Body Simulation (MBS) software which
+ is used for the dynamic analysis of any mechanical or mechatronic system. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCE_TAR_GZ]
+
+import os
+license_server = os.getenv('EB_SIMPACK_LICENSE_SERVER', 'license.example.com')
+license_server_port = os.getenv('EB_SIMPACK_LICENSE_SERVER_PORT', '000000')
```

Comment:
This is not used at all by the easyblock, you're just picking up `$EB_SIMPACK_LICENSE_SERVER` from the environment again there, so this can be removed?